test/unit/Exception/ModuleExceptionInterfaceTest.php

```diff
@@ -49,6 +49,12 @@ public function testCanBeCreated()
             'A valid instance of the test subject could not be created'
         );
 
+        $this->assertInstanceOf(
```

Comment:
So let me get this straight.

You're taking an interface, using a mock-hack to make it throwable, then asserting that the mock is throwable?

Sounds legit :rofl:

Reply:
🤷‍♂️